jetty-ee9/jetty-ee9-servlet/src/test/java/org/eclipse/jetty/ee9/servlet/FormTest.java

```diff
@@ -154,10 +150,15 @@ protected void service(HttpServletRequest request, HttpServletResponse response)
             })
             .send();
 
-        int expected = (maxFormContentSize != null && maxFormContentSize < 0)
-            ? HttpStatus.OK_200
-            : HttpStatus.BAD_REQUEST_400;
-        assertEquals(expected, response.getStatus());
+        assertEquals(expectedStatus, response.getStatus());
+    }
+
+    private AsyncRequestContent newContent(int size)
```

Comment:
I'd avoid this new method just to keep the code in sync with its ee10 counterpart.

Reply:
It is taken from the ee10 counterpart.